lib/spack/spack/cmd/info.py

```diff
@@ -48,11 +56,23 @@ def pad(string):
 
 def setup_parser(subparser):
     subparser.add_argument(
-        'name', metavar="PACKAGE", help="name of package to get info for")
+        'name', metavar='PACKAGE', help='name of package to get info for')
+
+
+def section_title(s):
+    return header_color + s + plain_format
+
+
+def version(s):
+    return spack.spec.version_color + s + plain_format
+
+
+def variant(s):
+    return spack.spec.enabled_variant_color + s + plain_format
 
 
 class VariantFormatter(object):
-    def __init__(self, variants, max_widths=(25, 20, 35)):
+    def __init__(self, variants, max_widths=(30, 20, 30)):
```

Comment:
Let's leave it for another PR, but [here it is](https://stackoverflow.com/questions/566746/how-to-get-linux-console-window-width-in-python)